managed/resource_telemetry_provider.go

```diff
@@ -0,0 +1,547 @@
+/*
+ * Copyright © 2022-present Yugabyte, Inc. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package managed
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/schemavalidator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	openapiclient "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+)
+
+type resourceTelemetryProviderType struct{}
```

Comment:
Seems a giant copy paste of my previous code for metrics exporter?

Reply:
It's the same thing. Telemetry providers are a general sink for audit logs, pg logs and metrics. Metrics exporter will be deprecated later on